config/settings.yml

```diff
@@ -80,6 +80,9 @@ feature_flags_enabled?: <%= !%w(test api-sandbox gamma).include?(ENV['ENV']) %>
 active_features:
   - :agfs_fee_reform
 
+# TODO: Update date to actual release date
+agfs_fee_reform_release_date: <%= Date.new(2018, 3, 1) %>
```

Comment:
Could this be abstracted to something like:
```ruby
<%= ENV['SCHEME_TEN_DATE'] || Date.new(2018, 4, 1) %>
```
So we could overwrite on different environments if needed?

Reply:
Definitely, I didn't go with that approach until we really needed, as it also requires updates on the deploy repo.